qftasm.js

```diff
@@ -176,15 +176,15 @@ function step_code() {
         vals.push(val);
     }
     
+    PC[1] = PC[1] + 1;
     PC[0] = PC[1];

```

Comment:
It's not always the same, PC[1] is changed (made different from PC[0]) when the program counter is written to.